.gitignore

```diff
@@ -26,5 +26,9 @@ node_modules
 
 # Users Environment Variables
 .lock-wscript
+<<<<<<< HEAD
 
 config.json
+=======
+config.json
+>>>>>>> server
```

Comment:
Looks like this is an unresolved merge conflict. Could you remove the green sections?